server/server.go

```diff
@@ -405,6 +405,7 @@ func (s *Server) setupConsensus() error {
 			Metrics:        s.serverMetrics.consensus,
 			SecretsManager: s.secretsManager,
 			BlockTime:      s.config.BlockTime,
+			BLS:            s.config.BLS,
```

Comment:
Why are we passing it as part of the `server` command configuration?

Can't we use it as part of the consensus `config` in the `genesis.json`?

That way `server` doesn't have to worry about whether BLS is enabled or not

Reply:
Resolving because this code has changed in the next PR https://github.com/0xPolygon/polygon-edge/pull/649.
Basically IBFT Hook module takes this configurations